### DISCUSSION TITLE
Increase assault rifle magazine capacity to 30

### DISCUSSION THREAD

**Discussion by Smrtonosan** [FakeFishGames/Barotrauma#13241] [Ideas and suggestions]
Modern standardized magazines have a capacity of 30.
For a heavy weapon you constantly run out of rounds with just 20 round magazines.
For comparison the SMG Magazines have 21, and the SMG can be held in the hotbar, meanwhile the assault rifle is a lot more clunky to use.

**Comment by DominO570** [comment]
Assault rifle pierces armor, gives a lot of bleeding and is very powerful even without taking into account talents, magazines can be stacked 2 pieces that probably reduces the inconvenience of reloading. 20 rounds is probably a normal amount considering.

**Reply by Smrtonosan** [reply]
Assault rifle is better than the SMG against armoured targets, but assault rifle magazines are more expensive compared to the SMG magazines, making it more punishing when missing or putting a couple of unnecessary or poorly aimed shots into your targets.

**Comment by bookslayer10** [comment]
Giving the Assault Rifle 50% more ammo capacity risks its power overlapping with the HMG. They are already both similar weapons, one of the ways the HMG is unique is because it has a higher ammo capacity.

Also, the Autoshotgun is in the same tier as the Assault Rifle, so buffing the Assault Rifle would likely make the Autoshogun underpowered and overshadowed.

**Reply by Smrtonosan** [reply]
Adding 10 rounds to the assault rifle magazine wouldn't change much other than cost effectiveness, especially once you consider that majority of players use keybinds to instantly reload their weapons.

**Reply by bookslayer10** [reply]
I don't think the Assault Rifle needs to be more cost effective, it's already pretty cheap to maintain, especially compared to the exotic ammo of other heavy weapons - fuel rods, explosive slugs, etc.

While keybinds are extremely effective, it still takes more concentration to constantly keep ammo topped off if a weapon has a lower mag size, especially with any type of fire rate buff.

The lower magazine size is a large part of what makes the Assault Rifle different from the HMG. With each shot costing more resources, time, and inventory space, as well as needing to manage the reloads much more often, it is sufficiently different to not overlap too much. Changing the Assault Rifle to have more ammo capacity would be a step backwards when it comes to the task of making the two weapons more unique.

**Reply by Smrtonosan** [reply]
AR is not comparable to the HMG
HMG has a higher rate of fire, higher DPS, higher penetration and same severance chance as the AR. The HMG can deal with swarms of both armoured and unarmoured enemies, all while being cost effective, while the AR cannot do either. From experience "everyone" switches to the HMG immediately once it's unlocked, negating the speed malus with talents, genes and stimulants. AR has more accuracy, and yet it doesn't matter as majority of engagements are close range against swarms of melee enemies. For spinelings the AR is even worse as they are hard to hit and each missed shot is 5% of your magazine gone. With 30 round magazines on the AR you'd be able to carry 60 rounds in a slot, while the HMG would still be able to carry 200 rounds. Even with 30 round magazines I'd still use the HMG as it is more fitting of a heavy weapon slot in terms of effectiveness. And is an actual upgrade over the SMG.

The HMG is cheap, the AR is not cheap to maintain. Yes weapons that are used for one-shotting enemies are going to have more expensive ammunition per shot. Regular shotgun shells aren't meant to be used against armoured targets.

**Reply by bookslayer10** [reply]
When I say "Comparable to the HMG," I mean they are similar weapons in concept. Both are fast firing magazine based weapons. Obviously the HMG is better than the Assault Rifle, the HMG is unlocked in a later talent, is more expensive to build, and takes more skills to use. The issue is the Assault Rifle becoming more like the HMG, which is poor game design.

If you want to buff the Assault Rifle, make it MORE distinct. Give it even higher base damage, maybe.

Remember that the HMG takes a lit of weapons skill to be able to even fire it, compared to how anyone can fire an Assault Rifle and have decent accuracy still. For any class other than Security Officer, the HMG won't be an option for a long time - that's where the AR comes in.
